code/modules/power/lighting.dm

```diff
@@ -277,6 +337,12 @@
 			removeStaticPower(static_power_used, STATIC_LIGHT)
 
 
+/obj/machinery/light/process()
```

Comment:
There are a LOT of lights on the station so process should be called only when it needs to be. i.e. charging or discharging. Use the /obj/machinery power procs/vars to help with that

Reply:
actually i'm full of shit, just set idle_power_usage properly when it's charging

Reply:
would this be accomplished by stopping process on initialize, then starting it again when it's needed? I don't know if machines are processed for any other reason so I'm unsure

Reply:
The internal cell gains power without using up any more than usual; think of it like it automatically storing away a bit of power whenever it uses the amount needed to power the light itself. This should be fine, I think?

Reply:
Eh, don't worry about it then